tasks/main.yml

```diff
@@ -97,6 +97,45 @@
   tags:
     - wg-config
 
+- block:
+    - name: Create private key for unmanaged hosts
+      shell: |
+        set -o errexit
+        set -o pipefail
```

Comment:
I suddenly started failing with a `Illegal option -o pipefail`. Adding an `executable: /bin/bash` fixed the issue for me.

Reply:
added